demo/docker/docker-compose-env.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.0'
+version: '3.7'
```

Comment:
Why you upgraded docker-compose version? We should have as lowest version as possible (until we use some features from higher version), because it cause that this docker-compose won't run with older dockers...